examples/next/pages/ui/components/storage/file-uploader/index.page.tsx

```diff
@@ -14,8 +14,7 @@ export default function FileUploaderEmail() {
       onSuccess={onSuccess}
       acceptedFileTypes={['image/*']}
       accessLevel="public"
-      hasMultipleFiles={true}
```

Comment:
This is actually a good example of why hasMultipleFiles is redundant. We shouldn't have to define both props here to get multi selection in the file uploader picker.